docs/reference/data-sources/push.md

```diff
@@ -0,0 +1,39 @@
+# Push source
```

Comment:
also I think you need to link to this `.md` file in `docs/SUMMARY.md`, otherwise it won't be surfaced

Reply:
done